Sources/WalletConnect/Session.swift

```diff
@@ -10,31 +10,30 @@ public struct Session {
     public let permissions: Permissions
     public let accounts: Set<Account>
     public let expiryDate: Date
+    public let blockchains: Set<String>
```

Comment:
What do you think of the idea of adding a CAIP-2 `Blockchain` type in the future, similar to `Account`? Does is make sense?